From b163380c13ca67c3270f516e2732e7f0ba12071e Mon Sep 17 00:00:00 2001
From: Nancy Chen <Nancy.Chen@freescale.com>
Date: Wed, 15 Aug 2012 20:53:15 -0500
Subject: ENGR00220497 [MX6Q, MX6DL]: Fix not able to set high bus freq

Fix not able to set high bus frequency from low bus frequency.

Signed-off-by: Nancy Chen <Nancy.Chen@freescale.com>
---
 arch/arm/mach-mx6/bus_freq.c |    1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/arm/mach-mx6/bus_freq.c b/arch/arm/mach-mx6/bus_freq.c
index 83310c4..95edf00 100644
--- a/arch/arm/mach-mx6/bus_freq.c
+++ b/arch/arm/mach-mx6/bus_freq.c
@@ -134,6 +134,7 @@ static void reduce_bus_freq_handler(struct work_struct *work)
 			clk_disable(pll2_400);
 
 		clk_disable(pll3);
+		med_bus_freq_mode = 0;
 	} else {
 		arm_mem_clked_in_wait = true;
 
-- 
1.7.9.5

